### PR TITLE
Grw-1786 Storyblok Tier Selector

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Make a storyblok component for Tier selector

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/TierSelector/TierSelector.tsx [hunk 1]
```diff
@@ -111,17 +111,17 @@ export const SecondaryTextStyle = styled.div(({ theme }) => ({
 export type TierItemProps = {
   title: string
   price: string
-  description: string
+  body: string
```
Comment (alebedev) [inline apps/store/src/components/TierSelector/TierSelector.tsx:114]: Nit: I'd expect this to accept blocks / rich text judging by the name.  Don't have a better idea for a name, though

File: apps/store/src/blocks/TierSelectorBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              <TierSelector.SecondaryTextStyle>
+                {selectedItem?.price}
+              </TierSelector.SecondaryTextStyle>
+            </>
+          ) : (
+            blok.title
+          )}
+        </TierSelector.HeaderWithTrigger>
+        <TierSelector.Content>
+          {tierSelectorItems.map((nestedBlock) => {
```
Comment (alebedev) [inline apps/store/src/blocks/TierSelectorBlock.tsx:34]: Mutating `nestedBlock` makes it hard to read.  You can try overriding `handleClick` instead
```
<TierSelectorItemBlock key={nestedBlock._uid} blok={nestedBlock} {...nestedBlock} handleClick={blockClickHandler(nestedBlock)} />
```

**Review by alebedev — APPROVED** [review]
Looks good.  Minor comments inside

**Comment by gustaveen** [comment]
Cool! 🙌 Just so I understand, is this an interim solution? Since the price and available tiers should be populated from the offer in the end right? @KarnellSchultz @alebedev

**Pull request closed without merging**